docs/src/content/docs/for-administrators/existing-preprocessing-pipelines.md

```diff
@@ -34,7 +34,6 @@ In the default configuration the pipeline performs:
 
 The pipeline also formats metadata fields:
 
--   **process date**: Takes a date string and returns a date field in the "%Y-%m-%d" format.
--   **parse timestamp**: Takes a timestamp e.g. 2022-11-01T00:00:00Z and returns that field in the "%Y-%m-%d" format.
+-   **parse timestamp**: Takes an ISO timestamp e.g. `2022-11-01T00:00:00Z` and returns that field in the `%Y-%m-%d` format.
```

Comment:
```
- ** parse_date_into_range**: Takes a string in the format `YYYY`, `YYYY-MM` or `YYYY-MM-DD` and an upper bound (`releaseDate`) and processes it into one of 3 arg output options: "dateRangeString" | "dateRangeLower" | "dateRangeUpper"

Reply:
I don't think we should have these Loculus duplicated docs at this point, one should just look at the code for these functions instead of having something that becomes stale with each change. Hence removing here.